src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -645,14 +676,36 @@ private void OnPreviewControlExpanded(object sender, EventArgs e)
             smallContentGrid.Visibility = System.Windows.Visibility.Hidden;
             BeginNextTransition(); // See if there's any more requests.
         }
-        
+
         private void OnPreviewControlCondensed(object sender, EventArgs e)
         {
             SetCurrentStateAndNotify(State.Condensed);
-            largeContentGrid.Visibility = System.Windows.Visibility.Hidden;
+            largeContentGrid.Visibility = Visibility.Hidden;
+            bubbleTools.Visibility = Visibility.Hidden;
             BeginNextTransition(); // See if there's any more requests.
         }
 
+        private void OnPreviewControlMouseMove(object sender, MouseEventArgs e)
+        {
+            if (IsMouseOver && IsCondensed && !IsInTransition)
```

Comment:
If the state is IsCondensed, then !IsTransition will be true. Do you need 2 checks for this?